lib/features/ui_kit/ui/ui_screen.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_learn_app/features/ui_kit/ui/cubit/ui_kits_cubit.dart';
+import 'package:flutter_learn_app/features/ui_kit/ui/cubit/ui_kits_state.dart';
+import 'package:flutter_learn_app/features/ui_kit/ui/selected_ui_kits.dart';
+import 'package:get_it/get_it.dart';
+
+class UIScreen extends StatelessWidget {
+  const UIScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
```

Comment:
No need for BlocProvider here as you provide in main.dart
just get from context.read<UiKitsCubit>().loadUiKits()